qa/workunits/rbd/concurrent.sh

```diff
@@ -88,9 +88,15 @@ function setup() {
 
 	# Set up some environment for normal teuthology test setup.
 	# This really should not be necessary but I found it was.
-	TOP="/tmp/cephtest"
+
+        if [ -f /etc/ceph/ceph.conf ]; then
+            TOP="/etc/ceph"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/ceph.client.0.keyring"
+        else
+            TOP="/tmp/cephtest"
+            export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
+        fi
 	export CEPH_ARGS="--conf ${TOP}/ceph.conf"
-	export CEPH_ARGS="${CEPH_ARGS} --keyring ${TOP}/data/client.0.keyring"
```

Comment:
@idryomov CEPH_ARGS is not passed here, even if its passed the old script just overwrites its own value of conf, Also I have not seen teuthology place ceph.conf in $HOME/cephtest , As @jdurgin mentioned it might not need old logic at all and the fix can be as simple as adding  " --name client.0" to CEPH_ARGS  , Please confirm.